src/mqt/core/__init__.py

```diff
@@ -2,7 +2,24 @@
 
 from __future__ import annotations
 
-from ._core import add
+from ._core import (
+    CompoundOperation,
+    Control,
+    Expression,
+    OpType,
+    Permutation,
+    QuantumComputation,
+    StandardOperation,
```

Comment:
This doesn't seem to expose all relevant classes.

Reply:
adjusted in new version